src/pages/Send/components/RecipientPicker/Tabs/Empty.js

```diff
@@ -14,30 +14,37 @@ type Props = {|
   tab: Tab,
 |}
 
+const emptyContactsTab = () => (
```

Comment:
Actually, you can use it without calling the function. See example here:

https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/pages/ErrorUnexpected/ErrorUnexpected.js#L21-L39